Solutions/Corvus.Azure.Gremlin.Tenancy/Corvus/Azure/GremlinExtensions/Tenancy/Internal/TenantGremlinContainerFactory.cs

```diff
@@ -219,7 +219,7 @@ private async Task<GremlinServer> CreateTenantGremlinServerAsync(GremlinConfigur
             string username = BuildTenantSpecificUserName(databaseName, containerName);
             if (string.IsNullOrEmpty(configuration.HostName) || configuration.HostName == DevelopmentHostName)
             {
-                return new GremlinServer(DevelopmentHostName, DevelopmentPort, true, username, DevelopmentAuthKey);
+                return new GremlinServer(DevelopmentHostName, DevelopmentPort, false, username, DevelopmentAuthKey);
```

Comment:
What is this change about?

Reply:
The code was utterly broken.

The intent of the code was to use the local emulator if you pass no hostname. But the effect was to use a URL where there was supposed to be a hostname (resulting in an unparsable URL), with the wrong port number, and requiring HTTPS even though the local emulator supports only HTTP.